### PR TITLE
[WIP] Add IIN Method document

### PR DESCRIPTION
**Pull request by ghoshbishakh**
In order to add the iin method in [DID Specification Registries](https://w3c.github.io/did-spec-registries/#did-methods), we need to define it in our own RFC and open a PR with its link.

### CODE REVIEW

File: rfcs/models/identity/iin_method.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+## Introduction
+
+An Interoperation Identity Network (IIN), is a system built on a distributed shared ledger that supports registration of DIDs to networks (e.g. permissioned consortium networks) and their units, as well as resolution of network DID documents form their DID. The IIN also has reputed trust anchors, which play the important role of certifying (or validating) the identity of owners of those DIDs, as well as their memberships in their respective networks, in effect creating roots of trust for cross-network interoperations. Thus, the IIN provides the foundation for our architecture to enable blockchain network-identity discovery and validation. The IIN DID Method specification conforms to the requirements specified in [Decentralized Identifiers (DIDs) v1.0](https://w3c.github.io/did-core/). The IIN as a system is detailed further [here](./iin.md).
+
+## IIN Method Syntax
+
+The IIN Method scheme MUST begin with the prefix `did:iin`, and is defined by the following ABNF based on the [DID Syntax](https://www.w3.org/TR/did-core/#did-syntax):
+
+```
+iin-did                = "did:iin" iin-name ":" iin-specific-id
```
Comment (VRamakrishna) [inline rfcs/models/identity/iin_method.md:22]: Can you align the equal signs vertically?

**Comment by VRamakrishna** [comment]
How (or how much) is this different from the spec in https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/rfcs/protocols/identity/readme.md?

**Comment by VRamakrishna** [comment]
Also, both this spec and the spec I linked to above (both of which likely should be merged) ideally belong to the `rfcs/format` folder as they contain precise structure specifications. For now, just leave it in place. I am working on another PR (partially done in my fork) where I have refactored the folder structure under `rfcs` to make it more navigable. I can pull your latest changes and move files around before submitting it.

**Comment by ghoshbishakh** [comment]
TODOS:
Fix title - DLT Interoperability DID Method ?
Fix location of file.
Fix the did method - iin ?


Eliminate overlap with rest of the rfc.

**Comment by ghoshbishakh** [comment]
@VRamakrishna one test is failing. Do I need to do anything about that?

**Comment by VRamakrishna** [comment]
@sanvenDev I see the Asset Exchange test failing in compilation with `Could not find com.weaver.corda.app.interop:interop-contracts:1.2.4-alpha.8`. Has that package been deleted?

**Comment by sanvenDev** [comment]
> @sanvenDev I see the Asset Exchange test failing in compilation with `Could not find com.weaver.corda.app.interop:interop-contracts:1.2.4-alpha.8`. Has that package been deleted?

This is actually a bug in corda testnet, need to fix it. If you want, you can ignore that workflow result now. In my or Krishna's next PR we can fix it.

**Comment by ghoshbishakh** [comment]
1. rebase
2. update rfc table
3. change it to formats

**Pull request closed without merging**